### ISSUE TITLE
Smoke FX Sideways on cannon fire

### ISSUE THREAD

**Issue by VenomousRequiem** [nrlulz/ACF#233]
When you fire the cannons, the smoke flies out up and down instead of left and right, as the muzzle breaks should probably make it do. It's nothing serious, just aesthetically displeasing.

**Comment by Amplar** [comment]
Don't visclip your gun. This is what causes the issue.

**Comment by VenomousRequiem** [comment]
No, sir, it's always done this. I've never visclipped a gun in fear that it would cause bugs.
